Procfile

```diff
@@ -0,0 +1,2 @@
+web: bundle exec puma -C config/puma.rb
```

Comment:
I think you can drop the `-C config/puma.rb` since that's already pma's default.